hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/security/TestRouterSecurityManager.java

```diff
@@ -259,6 +272,39 @@ private static String[] getUserGroupForTesting() {
     return groupsForTesting;
   }
 
+  @Test
+  public void testGetTopTokenRealOwners() throws Exception {
+    // Create conf and start routers with only an RPC service
+    Configuration conf = initSecurity();
+
+    Configuration routerConf = new RouterConfigBuilder()
+        .metrics()
+        .rpc()
+        .build();
+    conf.addResource(routerConf);
+
+    Router router = initializeAndStartRouter(conf);
+
+    // Create credentials
+    UserGroupInformation ugi = UserGroupInformation.createUserForTesting("router", getUserGroupForTesting());
+    Credentials creds = RouterSecurityManager.createCredentials(router, ugi, "some_renewer");
```

Comment:
assigning to ``creds`` is not required maybe

Reply:
@ayushtkn If we remove it, then running testGetTopTokenRealOwners alone is failing as its not able to find any token

Reply:
only doing this
```
 RouterSecurityManager.createCredentials(router, ugi, "some_renewer");
```
instead of 
```
Credentials creds = RouterSecurityManager.createCredentials(router, ugi, "some_renewer");
```

creates issues, it is just not assigning to creds, because we don't use ``creds`` variable anywhere